connectors/es/document.py

```diff
@@ -46,3 +49,27 @@ async def reload(self):
         self._seq_no = doc_source.get("_seq_no")
         self._primary_term = doc_source.get("_primary_term")
         self._source = doc_source.get("_source", {})
+
+    def debug(self, msg, *args, **kwargs):
```

Comment:
Personally I would prefer to name this methods `log_debug`, `log_info`, etc. . IMO it could be confusing seeing a method call like `sync_job.info(...)`, `connector.info(...)`, which someone could misinterpret. With `sync_job.log_info(...)` and `connector.log_info(...)` it's more obvious that the method call triggers a log. Especially for `sync_job.error()`, `connector.error()` these methods had a different meaning, right? (setting the error field).

WDYT?

Reply:
Make perfect sense. I will update.

Reply:
Done in https://github.com/elastic/connectors-python/pull/806/commits/1d89963fe2b4e40d3018fafa15de3b2ea3df8d62